code/API_definitions/qod-api.yaml

```diff
@@ -8,7 +8,7 @@ info:
   license:
     name: Apache 2.0
     url: https://www.apache.org/licenses/LICENSE-2.0.html
-  version: 0.8.0
+  version: 0.9.0
```

Comment:
Don't change the version number in the PR, we will define it when releasing a version to be used (most probably this change will go into v0.8.1).

Reply:
Ok.. Should I keep 0.8.0 or make it 0.8.1..?

Reply:
keep 0.8.0

Reply:
ok